py3status/module.py

```diff
@@ -337,11 +337,17 @@ def set_module_options(self, module):
 
         separator = fn(self.module_full_name, "separator")
         if not hasattr(separator, "none_setting"):
-            if not isinstance(separator, bool):
-                err = "Invalid `separator` attribute, should be a boolean. "
-                err += f"Got `{separator}`."
-                raise TypeError(err)
-            self.i3bar_module_options["separator"] = separator
+            # HACK: separator is a valid setting in the general section
```

Comment:
That's interesting, I did not recall that at all...

Reply:
Quick glance.. Maybe things are a bit hazy, but it looks like there could be two separator configs.

- (General) String `separator` https://i3wm.org/i3status/manpage.html#_general
- (Module) Boolean `separator` https://i3wm.org/i3status/manpage.html#_universal_module_options

... both may gotten merged... Putting `separator = True` in py3status config section should turn them all off.

Maybe we need two different `separator` coding.... One for general (string) and one for py3status (boolean).